src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1

```diff
@@ -64,7 +64,7 @@ $commandOptions = @{
   sync = "--output-directory='' --id='' --package-id='' -?" + $allcommands
   optimize = "--deflate-nupkg-only --id='' -?" + $allcommands
   export = "--include-version-numbers --output-file-path='' -?" + $allcommands
-  template = "--name=''" + $allcommands
+  template = "--name='' --include-remembered-arguments" + $allcommands

```

Comment:
Shouldn't this be attached to the export command, and not the template command?

Reply:
Fixed